Portal/src/Datahub.Portal/Pages/Project/DataProject/ProjectInfo.razor

```diff
@@ -42,7 +42,7 @@
         </MudBadge>
     }
 
-    @if (IsProjectAdmin || IsDatahubAdmin)
+    @if ((IsProjectAdmin || IsDatahubAdmin) && Project?.MetadataAdded != true)
```

Comment:
Do we want this? They won't be able to get to the metadata page w/o manually entering the URL

cc @ErikApption